### PR TITLE
No nonsense explanation of compiler options

### PR DESCRIPTION
**Pull request by fulldecent**

### CODE REVIEW

File: docs/installing-solidity.rst [hunk 1]
```diff
@@ -18,30 +18,38 @@ will use the latest release.
...
+on your computer. Choose a commandline compiler if you are working on a larger contract
+or if you require more compilation options.
 
 npm / Node.js
 =============
 
-This is probably the most portable and most convenient way to install Solidity locally.
+Use `npm` for a convenient and portable way to install `solcjs`, a Solidity compiler. The
+`solcjs` program has less features than all options further down this page. Our 
+`Using the compiler <http://solidity.readthedocs.io/en/develop/using-the-compiler.html>`
```
Comment (chriseth) [inline docs/installing-solidity.rst]: Please use proper RST links for relative references.

File: docs/installing-solidity.rst [hunk 2]
```diff
@@ -18,30 +18,38 @@ will use the latest release.
...
+documentation assumes you are using the full-featured compiler, `solc`. So if you if you
+install `solcjs` from `npm` then you're on your own.
```
Comment (chriseth) [inline docs/installing-solidity.rst]: There is some documentation at https://github.com/ethereum/solc-js, though.

File: docs/installing-solidity.rst [hunk 3]
```diff
@@ -18,30 +18,38 @@ will use the latest release.
...
 
-A platform-independent JavaScript library is provided by compiling the C++ source
-into JavaScript using Emscripten. It can be used in projects directly (such as Remix).
+Note: The `solc-js <https://github.com/ethereum/solc-js>` project is derived from the C++
+`solc` by using Emscripten. `solc-js` can be used in projects directly (such as Remix).
```
Comment (chriseth) [inline docs/installing-solidity.rst]: Perhaps more specifically: in javascript projects

**Review by chriseth — CHANGES_REQUESTED** [review]
Thanks a lot! Just some tiny changes and this can go in!

**Comment by fulldecent** [comment]
@chriseth, you got it

**Pull request merged**